src/bench/prevector.cpp

```diff
@@ -62,6 +67,24 @@ static void PrevectorResize(benchmark::State& state)
     }
 }
 
+template <typename T>
+static void PrevectorDeserialize(benchmark::State& state)
+{
+    CDataStream s0(SER_NETWORK, 0);
+    prevector<28, T> t0;
+    t0.resize(28);
+    for (auto x = 0; x < 1001; ++x) {
+        s0 << t0;
+    }
```

Comment:
Self review: To closing to actual condition, it would be better to include the case where the size exceeds the initial capacity.

Reply:
done.